### PR TITLE
Markdown pro lekci o slovnících

### PR DESCRIPTION
**Pull request by encukou**
* Začlenění commitů z pyladies.cz do zbytku materiálů
* Převedení slovníků, JSON a GitHub API na Markdown

### CODE REVIEW

File: lessons/projects/asteroids/index.html [hunk 1]
```diff
@@ -64,7 +63,9 @@
                             prvků a každý prvek v ní může být pouze jednou.
                             (Na množinu se dá dívat i jako na slovník bez
                             hodnot.)
-                            Pythoní dokumentace obsahuje k množinám
+                            Je k dispozici
+                            <a href="https://github.com/pyvec/cheatsheets/blob/master/sets/sets-cs.pdf">tahák na množiny</a>, a
+                            pythonní dokumentace obsahuje k množinám
                             <a href="https://docs.python.org/3/tutorial/datastructures.html#sets">tutoriál</a> i
                             <a href="https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset">podrbný popis</a>.
```
Comment (frenzymadness) [inline lessons/projects/asteroids/index.html]: s/podrbný/podrobný/

File: lessons/projects/asteroids/index.html [hunk 2]
```diff
@@ -274,29 +299,33 @@
                             teď jen zkopíruj:
 <pre>
 def distance(a, b, wrap_size):
+    """Distance in one dirextion (x or y)"""
```
Comment (frenzymadness) [inline lessons/projects/asteroids/index.html]: s/dirextion/direction/

File: lessons/projects/asteroids/index.html [hunk 3]
```diff
@@ -274,29 +299,33 @@
...
     result = abs(a - b)
     if result > wrap_size / 2:
         result = wrap_size - result
     return result
 
 def overlaps(a, b):
+    """Returns true iff two space objects overlap"""
```
Comment (frenzymadness) [inline lessons/projects/asteroids/index.html]: s/iff/if/ and maybe s/true/True/

File: lessons/projects/github-api/index.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    token = soubor.read().strip()
+
+headers = {'Authorization': 'token ' + token}
+
+stranka = requests.get('https://api.github.com/user', headers=headers)
+stranka.raise_for_status()
+print(stranka.text)
+```
+
+Co se stalo? Tím, že jsi Githubu dal{{a}} svůj token,
```
Comment (frenzymadness) [inline lessons/projects/github-api/index.md]: Why there isn't used function `gnd()`?

Comment (encukou) [inline lessons/projects/github-api/index.md]: `{{a}}` is a shortcut for `{{gnd('', 'a'}}`. It's over two times more frequent than other uses of `gnd`.

File: lessons/projects/asteroids/index.html (hunk 3 above)
Comment (encukou) [inline lessons/projects/asteroids/index.html]: Both of these are intentional, though subtle.
[iff](https://en.wikipedia.org/wiki/Iff) stands for “if and only if”.
“true” stands for any value for which `bool(value) is True`.

**Pull request merged**